tools/c7n_gcp/tests/test_dataproc.py

```diff
@@ -0,0 +1,21 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from gcp_common import BaseTest
+
+
+class TestDataprocCluster(BaseTest):
+
+    def test_query(self):
+        project_id = 'cloud-custodian'
+        factory = self.replay_flight_data('test_dataproc_clusters_query',
+                                          project_id=project_id)
+        p = self.load_policy(
```

Comment:
we really need a testing helper here, to set the applicable regions to the ones desired to cut out the extraneous output recording. adding a mechanism for such to c7n_gcp/region.py:Region would be useful

Reply:
I've gone ahead and a test fixture helper here for setting only required regions when using region as a parent, and removed the 20+ extraneous test files